index.js

```diff
@@ -405,7 +402,7 @@ const ukLocalAuthorities = [
 {value:"Antrim & Newtownabbey Borough Council",label:"Antrim & Newtownabbey Borough Council"},
 {value:"Ards and North Down Borough Council",label:"Ards and North Down Borough Council"},
 {value:"Armagh City",label:"Armagh City"},
-{value:" Banbridge and Craigavon Borough Council",label:" Banbridge and Craigavon Borough Council"},
+{value:"Banbridge and Craigavon Borough Council",label:"Banbridge and Craigavon Borough Council"},
```

Comment:
Should this be `Armagh City, Banbridge and Craigavon District Council`?

Reply:
Removed Armagh City
Amended to Armagh City, Banbridge and Craigavon.